src/cdk-experimental/listbox/listbox.ts

```diff
@@ -194,360 +158,382 @@ export class CdkOption<T = unknown> implements ListKeyManagerOption, Highlightab
     'role': 'listbox',
     'class': 'cdk-listbox',
     '[id]': 'id',
-    '(focus)': '_focusActiveOption()',
-    '(keydown)': '_keydown($event)',
-    '[attr.tabindex]': '_tabIndex',
+    '[attr.tabindex]': 'disabled ? null : 0',
     '[attr.aria-disabled]': 'disabled',
     '[attr.aria-multiselectable]': 'multiple',
     '[attr.aria-activedescendant]': '_getAriaActiveDescendant()',
     '[attr.aria-orientation]': 'orientation',
+    '(focus)': '_handleFocus()',
```

Comment:
I see that we're using `focusout` below to handle blurring. Should we use `focusin` here as well? Focus could land directly on the options if the user clicks on them, rather than tabbing into it.

Alternatively, we could use the `FocusMonitor` with deep monitoring enabled.

Reply:
Here's we're using focus to handle a focus on the listbox element specifically, so we can redirect the focus to one of the options.